becquerel/core/energycal.py

```diff
@@ -1,7 +1,7 @@
 """"Energy calibration classes"""
 
 from abc import ABCMeta, abstractmethod, abstractproperty
-from builtins import dict, super, zip  # pylint: disable=redefined-builtin
+from future.builtins import dict, super, zip
```

Comment:
Are we planning to add this `pylint` disable to the `pylintrc`

Reply:
They have been there for a while. However, pylint only doesn't like the import if it comes from `builtins`. At least on my system it did not complain about those includes from `future.builtins`, thus I removed them.